libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.ts

```diff
@@ -24,9 +24,11 @@ import {
 } from '@energinet-datahub/watt/drawer';
 import { WattButtonModule } from '@energinet-datahub/watt/button';
 import { UserOverviewItemDto } from '@energinet-datahub/dh/shared/domain';
+import { WattModalModule } from '@energinet-datahub/watt/modal';
```

Comment:
nitpick: this module is not needed here since the modal is not used in this component directly but in `dh-edit-user-modal.component.component.ts`, which imports `WattModalModule` as well.